### ISSUE TITLE
RFC: remove support for ranges as a commit option

### ISSUE THREAD

**Issue by MasterDuke17** [perl6/whateverable#346]
I don’t think it (e.g., 1218.12..HEAD) has ever been used in the wild so it’s just an extra code path to maintain and test.

**Comment by AlexDaniel** [comment]
I agree. Personally I did use it a few times (it was something like `2018.10..HEAD` when there were not many commits after the release, also I remember doing something like 2018.10~100..2018.10), but nowadays I'd much rather use Whateverable as a library and run it on my own computer ([gist](https://gist.github.com/AlexDaniel/88b38af5db9adcf0b711ac3df31dd431)). Feel free to remove the feature and the corresponding tests.
